wiki/Beatmap/Pattern/common/double/en.md

```diff
@@ -0,0 +1,20 @@
+---
+stub: true
+tags:
+  - doubles
+---
+
+# Doubles
+
+A **Double** pattern is made when 2 [hit objects](/wiki/Gameplay/Hit_object) occur after each other in a 1/2 or faster rythm spacing.
+
+![](img/double.jpg "Visual representation of a double in the osu! gamemode.")
+
+In the [osu!](/wiki/Game_mode/osu!) [gamemode](/wiki/Game_mode), doubles are when 2 [hit circles](/wiki/Gameplay/Hit_object/Hit_circle) are [stacked](/wiki/Beatmapping/Mapping_techniques/Stack) on eachother.
+
+![](img/maniadouble.png "Visual representation of a double in the osu!mania gamemode.")
+
+In the [osu!mania](/wiki/Game_mode/osu!mania) [gamemode](/wiki/Game_mode), Doubles are known as [Mini jacks](/wiki/Beatmap/Pattern/osu!mania/Jack)
```

Comment:
not entirely true; double typically refers to a jump (2-chord) instead of mini jacks

![image](https://github.com/user-attachments/assets/060e0126-2290-41d3-9923-765a4e9ac28e)

Reply:
The reason i linked mini jacks because at first doubles were supposed to be a osu! pattern rather than a one common for all modes, and mini jacks reminded me from doubles from !std. will fix now

Reply:
I just realised that I didn't change the pattern visual, will do that when possible

Reply:
Fixed